saltgui/static/stylesheets/keys.css

```diff
@@ -0,0 +1,3 @@
+.fingerprint {
+  font-family: monospace;
```

Comment:
Looks good. One think (at least in my browser) text is vertically align to the top. It looks much better if we will add `vertical-align: middle;`. What do you think.

Btw this I believe is common issue:
![zrzut ekranu 2019-02-10 o 15 55 25](https://user-images.githubusercontent.com/5910851/52535258-5480c380-2d4c-11e9-83cb-71423989176b.png)

Reply:
Same here. The solution affects the placement of some elements in situation with higher rows. Such as when viewing grain values.